docs/source/models/gbd2017/causes/measles/index.rst

```diff
@@ -83,6 +83,31 @@ Modeling Measles in GBD 2017
 
    Make sure to check on measles sequelae as well. Our models so far have not paid much attention to the nonfatal side, but it looks like some of the complications can persist well after someone recovers from measles, so maybe that's important to think about.
 
+GBD Hierarchy
+-------------
+
+**Hierarchy table:**
+
++----------------------------------+------------+-----------+
+| **Cause**                        | **GBD ID** | **Level** |
++----------------------------------+------------+-----------+
+| All causes                       | 294        | 0         |
++----------------------------------+------------+-----------+
+| Communicable, maternal, neonatal | 295        | 1         |
+| and nutritional diseases         |            |           | 
++----------------------------------+------------+-----------+
+| Other infectious diseases        | 961        | 0         |
++----------------------------------+------------+-----------+
+| Measles                          | 341        | 3         |
++----------------------------------+------------+-----------+
+
+- {Measles} = [Moderate Measles, Severe Measles]
```

Comment:
Let's make it clearer that this is about the sequelae of measles, which we use only for calculating disability weights

Reply:
@kiranklc Somewhere I would also quote the YLD appendix where they say they assume 50% of measles cases are moderate and 50% are severe. I don't remember seeing any data backing that up (though if we could find some, that would be good) - we should clearly indicate that this is an assumption of our model, based on what they did in the GBD.

Reply:
@NathanielBlairStahn Sure. I will add that below the hierarchy diagram.